app/models/user.rb

```diff
@@ -86,6 +86,14 @@ def active_profile
     profiles.find(&:active?)
   end
 
+  # This user's most recently activated profile that has also been deactivated
+  # due to a password reset, or nil if there is no such profile
+  # @return [Profile,nil]
```

Comment:
Yes I can see how that would be valuable. It is not a style we are using in this project, though, as far as I know. 

I tend to depend on the tests for method documentation. If a method is still unclear when it is well-tested, generally I think that means that it needs so refactoring attention.